0x09_finished/app/resources/post.py

```diff
@@ -27,23 +29,30 @@ class PostResource(Resource):
     def get(self, post_id):
         schema = PostSchema()
         post = Post.query.get_or_404(post_id)
-        return {"post": schema.dump(user).data}
+        return {"post": schema.dump(post)}
 
     def put(self, post_id):
         schema = PostSchema(partial=True)
         post = Post.query.get_or_404(post_id)
-        post, errors = schema.load(request.json, instance=post)
-        if errors:
-            return errors, 422
+        try:
+            update_value = schema.load(request.json, instance=post)
+            Post.query.filter_by(id=post_id).update(update_value)
+        except ValidationError:
+            return 422
```

Comment:
[Marshmallow.Schema.load](https://marshmallow.readthedocs.io/en/stable/api_reference.html#marshmallow.Schema.load)
- Changed in version 3.0.0b7: This method returns the deserialized data rather than a (data, errors) duple. A ValidationError is raised if invalid data are passed.

Reply:
![image](https://user-images.githubusercontent.com/52899668/104472767-418d6280-55f7-11eb-9413-7324d3dd2bce.png)